.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on: 
+  push: 
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+  
+    - name: update rubygems
+      run: gem update --system
+
+    - name: Install onetimepass
+      run: pip install onetimepass
+
+    - name: Configure gem credentials
+      run: |
+        echo "GEM_HOST_API_KEY=${{ secrets.RUBYGEMS_API_KEY }}" >> $GITHUB_ENV
+        echo "RUBYGEMS_MFA_KEY=${{ secrets.RUBYGEMS_MFA_KEY }}" >> $GITHUB_ENV
+
+    - name: Build newrelic-telemetry_sdk gem
+      run: gem build newrelic-telemetry_sdk.gemspec
+
+    - name: Determine version
+      run: echo "VERSION=$(ls newrelic-telemetry_sdk-*.gem | ruby -pe 'sub(/newrelic-telemetry_sdk\-(.*).gem/, "\\1")')" >> $GITHUB_ENV
```

Comment:
Does this actually work?  Compared to the Ruby Agent's workflow: 
`echo ::set-env name=VERSION::$(ls newrelic_rpm-*.gem | ruby -pe 'sub(/newrelic_rpm\-(.*).gem/, "\\1")')`

We're calling `echo ::set-env name=VERSION` so doesnt't the above just echo the version to STDOUT?

Reply:
Actually, it's echoing into $GITHUB_ENV which is the new preferred way to set environment variables. I tested this part on my fork [here](https://github.com/tannalynn/newrelic-telemetry-sdk-ruby/runs/1322436764?check_suite_focus=true).